tests/snapshots/tests.contrib.wsgi.test_wsgi.test_wsgi_base_middleware_500.json

```diff
@@ -0,0 +1,43 @@
+[[
```

Comment:
Note - The request span modifier was not called. I believe this is a bug in the existing wsgi middleware. This follow up change should fix it: https://github.com/DataDog/dd-trace-py/compare/munir/refactor-wsgi-middleware...munir/wsgi-bugfix-request-span-modifiers?expand=1.

Reply:
This is where we set the resource name and tags on the wsgi request span: https://github.com/DataDog/dd-trace-py/blob/1.2.1/ddtrace/contrib/wsgi/wsgi.py#L135-L144.  If an exception is raised [here](https://github.com/DataDog/dd-trace-py/blob/1.2.1/ddtrace/contrib/wsgi/wsgi.py#L133) or [here](https://github.com/DataDog/dd-trace-py/blob/1.x/ddtrace/contrib/wsgi/wsgi.py#L124) then the resource names and request tags will not be set. In a follow PR my place is move the "span_modifers" to this [line](https://github.com/DataDog/dd-trace-py/blob/1.x/ddtrace/contrib/wsgi/wsgi.py#L122)